components/extractors/marketing-sales/bing-ads/index.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Microsoft Advertising (Bing Ads)
+permalink: /components/extractors/marketing-sales/bing-ads/
+redirect_from:
+    - /extractors/marketing-sales/bing-ads/
+---
+
+* TOC
+{:toc}
+
+The Microsoft Advertising (Bing Ads) extractor supports extracting either campaign entity data or various types of reports available. In case of reports you can specify your own set of columns and primary key to use in Keboola Storage, but there are also presets of columns and appropriate primary keys available.
+
+## Prerequisites
+<!-- TODO: Please check this is a correct procedure. I could not try doing this in the production environment myself. - F. Navrkal -->
+First, you must create a Developer Token. To do so, please follow [this guide in the official documentation](https://learn.microsoft.com/en-us/advertising/guides/get-started?view=bingads-13#get-developer-token).
+
+Then you must find your Bing Ads Account ID and Customer ID. Once again, please follow [this part of the official documentation](https://learn.microsoft.com/en-us/advertising/guides/get-started?view=bingads-13#get-ids). Both should be numbers such as `391827251`.
```

Comment:
Someone with access to a Microsoft Advertising (Bing Ads) production environment should test this.

Reply:
This is correct, it has been tested many times by our clients who us it